spec/visual/samples/nasm

```diff
@@ -1,9 +1,15 @@
-;Simple test of NASM parser
+; Simple test of the NASM parser
+;
+; Sample compilation (requires 32-bit support from NASM and GCC).
```

Comment:
I'm not sure I understand what this adds. The purpose of the visual sample is to check the lexer is working correctly. There is already sufficient checking of the commenting syntax.

Reply:
I understand you. However, I'd argue most on a Ruby project do not know NASM syntax or how to test if a NASM file is a valid program that is working as expected. Those two lines are enough to test if the contents of the example are syntactically valid and semantically sound.

However, I am willing to remove these extra lines if they seem excessive to you.

Reply:
That's true but it's also true of most languages for which we have lexers. It's still the case that we have many visual samples that don't meet this criterion so I apologise if it seems arbitrary but ideally I am trying to move visual samples to be more targeted examples that contain less duplication. If you could remove these lines, that'd be much appreciated.

Reply:
> (...) if it seems arbitrary (...)

It does not, and your point makes sense as the other visual samples really do not have instructions.

I have removed these lines.

Reply:
Thanks!